### PR TITLE
Store the form data as JSON rather than as serialized JSON into string.

### PR DESCRIPTION
**Pull request by ynedelchev**
### Description

When Custom Forms are pulled from Aria Automation, they are stored on the file system (the repo) in a form similar to 
```
	 {
	    "id": "e694a748-7067-47d1-91a4-614da73dda03",
	    "name": "Test",
	    "form": "{\"layout\": {...},\"schema\": {...},\"options\": {...}}",
	    "styles": null,
	    "sourceType": "com.vmw.blueprint",
	    "sourceId": "71ac6ebc-6a94-3c5a-8c00-2a44ddf81bce",
	    "type": "requestForm",
	    "status": "ON",
	    "formFormat": "JSON"
	 }
```
Here, please note that the form field is a double serialized JSON as string : "{\"layout\": {...},\"schema\": {...},\"options\": {...}}"
which makes it goes in one line and very difficult for a human to work with. If there is any commits and. changes in the form in the repo, the diffs are very difficult to find (when reviewing pull requests). 
As a whole it is not human friendly and very difficult for a human to deal with. 

The current pull request, makes it so that the format will become a properly formatted JSON object like: 
```
{
	    "id": "e694a748-7067-47d1-91a4-614da73dda03",
	    "name": "Test",
	    "form": {
	      "layout": {
	      ...
	      },
	      "schema": {
	      ...
	      },
	      "options": {
	      ...
	      }
	    },
	    "styles": null,
	    "sourceType": "com.vmw.blueprint",
	    "sourceId": "71ac6ebc-6a94-3c5a-8c00-2a44ddf81bce",
	    "type": "requestForm",
	    "status": "ON",
	    "formFormat": "JSON"
}
```
This way it is more easy to work with. 
### Checklist

- [X] I have added relevant error handling and logging messages to help troubleshooting
- [x] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation, relevant usage information (if applicable)
- [x] I have updated CHANGELOG.md with a short summary of the changes introduced
- [X] I have tested against live environment, if applicable
- [x] I have synced any structure and/or content vRA-NG improvements with vra-ng and ts-vra-ng archetypes (if applicable) - 
- [X] I have my changes rebased and squashed to the minimal number of relevant commits. **Notice: don't squash all commits**
- [X] I have added a descriptive commit message with a short title, including a `Fixed #XXX -` or `Closed #XXX -` prefix to auto-close the issue - NOT APPLICABLE

### Testing

On Actual Aria Automation installation (on the lab)
- Pull content to some VRA-NG Project that contains Catalog ItemsCustom Forms.
- Check that the custom forms metadata file in folder `catalog-item/forms` contain two JSON files. 
  * one with extension `-FormData.json` that contains just the Form Data in JSON format - formated.
  * another one without the `-FormData` suffix. This one contains all data about the custom form including metadata, but also the form itself in JSON format. The form should exist as a properly formatter JSON object and not as a string as it was before.
- Once the content is pulled and verified, push it back to make sure it is pushable.

### Release Notes

Export (Pull) Arai Automation Custom forms in a structured JSON, rather than in string encoded JSON, when stored together with the Custom Form metadata, thus making it more human friendly to operate with when stored in a repo.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 * [artifact-manager] IAC-796 / Upgrade VCD archetype to support Angular 15
 * [vRA-NG] IAC-782 / vra-ng:pull : Added support for export of policies of the following types: Approval Policy, Day 2 Actions Policy, Deployment Limit Policy, Lease Policy, Resource Quota policy.
 * [artifact-manager] IAC-782 / vrealize:push : Added support for import of policies of the following types: Approval Policy, Day 2 Actions Policy, Deployment Limit Policy, Lease Policy, Resource Quota policy.
+* [artifact-manager] Export (Pull) Arai Automation Custom forms in a structured JSON, rather than in string encoded JSON, when stored together with the Custom Form metadata, thus making it more human friendly to operate with when stored in a repo.
```
Comment (VenelinBakalov) [inline CHANGELOG.md:19]: That looks great, could you also update the Release.md file

Comment (ynedelchev) [reply CHANGELOG.md:19]: Done

File: common/artifact-manager/src/test/java/com/vmware/pscoe/iac/artifact/store/vrang/VraNgCustomFormTest.java [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        VraNgCustomFormAndData repoFormSource =
+        new VraNgCustomFormAndData("123",
+            "Some Name", null, null,
+            "com.vmw.vro.workflow", "requestForm",
+            "requestForm", "ON", "JSON");
+
+        repoFormSource.setForm(jsonString);
+        JsonElement form = repoFormSource.getForm();
+
+        new VraNgCustomForm(repoFormSource);
```
Comment (akantchev) [inline common/artifact-manager/src/test/java/com/vmware/pscoe/iac/artifact/store/vrang/VraNgCustomFormTest.java:158]: I think this constructor here is not needed (as it is not used anywhere).

Comment (ynedelchev) [reply common/artifact-manager/src/test/java/com/vmware/pscoe/iac/artifact/store/vrang/VraNgCustomFormTest.java:158]: As this is a test, my idea was to invoke the constructor in order to verify (assert) that it does not throw an exception. 

Changing it to 
```
assertDoesNotThrow(() -> new VraNgCustomForm(repoFormSource));
```
and moving it into the `//THEN` section of the test for more clarity.

**Review by akantchev — APPROVED** [review]

**Review by akantchev — APPROVED** [review]

**Pull request merged**